ziti/edge/channel.go

```diff
@@ -16,7 +16,7 @@ const (
 func NewBaseSdkChannel(underlay channel.Underlay) *BaseSdkChannel {
 	senderContext := channel.NewSenderContext()
 
-	defaultMsgChan := make(chan channel.Sendable, 4)
+	defaultMsgChan := make(chan channel.Sendable, 64)
```

Comment:
why 64?

Reply:
Testing showed lots of drops of acks, so I bumped this up to match what we use for links. This resolved the issue.